content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -76,6 +76,12 @@ Disables all CLI warnings generated by `logger.warn`.
 
 To configure `prisma generate` to [generate a Prisma Client for Data Proxy](/data-platform/classic-projects/data-proxy) on any execution, even when `--data-proxy` is not explicitly set for the CLI command, set the value of this environment value to a truthy value (e.g. `PRISMA_GENERATE_DATAPROXY=true`).
 
+<Admonition>
+
+In Prisma 5.2 and higher, `--accelerate` and `--data-proxy` are no longer strictly required to use Accelerate or Data Proxy. However, if these flags are omitted, a query engine will still be included in the generated client. To avoid downloading any engine, you can use the [`--no-engine` option](./command-reference) when generating your client. This is advised when you are deploying to an environment where application size is critical, such as in serverless or edge functions.
```

Comment:
```suggestion
In Prisma 5.2.0 and higher, `--accelerate` and `--data-proxy` are no longer strictly required to use Accelerate or Data Proxy. However, if these flags are omitted, a query engine will still be included in the generated client. To avoid downloading any engine, you can use the [`--no-engine` option](./command-reference) when generating your client. This is advised when you are deploying to an environment where application size is critical, such as in serverless or edge functions.
```